CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ReplicatorConfigurationRequestHandler.swift

```diff
@@ -22,6 +22,68 @@ public class ReplicatorConfigurationRequestHandler {
         /////////////////////////////
             
         // TODO: Change client to expect replicator config, not the builder.
+        case "replicatorConfiguration_collection":
```

Comment:
'replicatorConfiguration_collection' looks like duplicating the 'replicatorConfiguration_configure' case, same configuration can be done via 'replicatorConfiguration_configure'. So there is no value addition here. Also these are all deprecated properties as well. For now, separating only AddCollection, AddCollections, RemoveCollection is also fine.

If you feel like, more modularizing, you could create a Jira ticket to cleanup `replicatorConfiguration_configure` and do it later.